plugins/modules/aci_l3out_floating_svi_secondary_ip.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_floating_svi_secondary_ip
```

Comment:
Did you consider combining the two secondary ip address modules?

Reply:
I think it would create a confusion due to different parents and naming of the files. In any case I went ahead with separate modules after discussions.

Reply:
Ok if discussed and considered then its good! Do think the difference is little (only a domain class (l3extRsDynPathAtt) insertion into the url with tDn.